### ISSUE TITLE
can xgboost support lambdamart currently or in future?

### ISSUE THREAD

**Issue by gusuperstar** [dmlc/xgboost#896]

**Comment by tqchen** [comment]
yes, see https://xgboost.readthedocs.org/en/latest/faq.html#do-you-support-lambdamart

**Comment by gusuperstar** [comment]
“rank:pairwise” –set XGBoost to do ranking task by minimizing the pairwise loss

do u mean this?  Since lambdamart is a listwise approach, how can i fit it ? including commond, parameters, and training data format. could u give a brief demo? many thanks

**Comment by lglhuada** [comment]
I am meeting similar issues like gusuperstar. What is the training data format when it comes to lambdamart with xgboost?

**Comment by lglhuada** [comment]
Sorry I did not check the tutorial carefully, some information from https://xgboost.readthedocs.io/en/latest/input_format.html#additional-information and https://github.com/dmlc/xgboost/tree/master/demo/rank might be helpful.
